test/Entropy.FunctionalTests/Entropy.FunctionalTests.csproj

```diff
@@ -12,6 +13,18 @@
     <RootNamespace>Entropy.FunctionalTests</RootNamespace>
   </PropertyGroup>
 
+  <PropertyGroup Condition=" '$(TargetFramework)' != 'netcoreapp2.0' ">
+    <!-- Work around https://github.com/dotnet/sdk/issues/926. Align with bitness of the web site projects. -->
+    <PlatformTarget>x86</PlatformTarget>
+
+    <!--
+      Work around https://github.com/Microsoft/vstest/issues/428 aka https://github.com/aspnet/Mvc/issues/5873.
```

Comment:
These workarounds should not be necessary now that we build with Microsoft.NET.Test.Sdk 15.3.0-*